lang/llvm-3.7/Portfile

```diff
@@ -72,6 +72,12 @@ if {${subport} eq "llvm-${llvm_version}"} {
     default_variants    +analyzer
 }
 
+if {${os.platform} eq "darwin" && ${os.major} < 14 && ${cxx_stdlib} eq "libc++"} {
+   # libxml2 depends on icu, which requires C++11
+   # see https://trac.macports.org/ticket/59391 for why this is needed on OS X Mavericks (10.9)
+   depends_lib-replace port:libxml2 port:libxml2-bootstrap
```

Comment:
Doesn't this need any changes to find libxml2-bootstrap?

Reply:
> Doesn't this need any changes to find libxml2-bootstrap?

I am sorry, but I do not understand your concern.
Why would there be difficulty in finding libxml2-bootstrap?

Reply:
Also the comment should probably mention C11/max_align_t since that's the real issue.

Reply:
> Why would there be difficulty in finding libxml2-bootstrap?

Because it installs in a different prefix. See llvm-3.4: https://github.com/macports/macports-ports/blob/master/lang/llvm-3.4/Portfile#L73-L75

Reply:
> Doesn't this need any changes to find libxml2-bootstrap?

Of course you are right.
During testing, the configure script was finding the **sytem** headers and libraries.
Thank you for noticing.

Reply:
> Also the comment should probably mention C11/max_align_t since that's the real issue.

I modified the comment and the commit message.
Hopefully, it is more clear.